css/style.css

```diff
@@ -1,19 +1,205 @@
+/* Hero */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+.hero {
+    background: url("../img/first-background.jpg");
+    height: 650px;
+    background-position: right 27% bottom 5%;
+    width: 100%;
+    flex-direction: column;
+}
+
+
+/* Hero Text */
+
+h1,
+.hero p {
+    color: white;
+}
+
+h1 {
```

Comment:
I noticed the text isn't aligned in the center on mobile viewport, I would add "text-align:center". 
We are not required to make it responsive, but thought I'd let you know :) 

Same goes for the <p> below header.

Reply:
I somehow missed this for the hero text, all other text are text aligned center, thanks for spotting this